tests/plan/export/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "cp data/plan.txt ${tmp}/plan.fmf"
```

Comment:
When `data/.fmf` is included in the commit `pushd data` is the only thing needed here.